### ISSUE TITLE
[BLE] bluetooth seeds encryption crashed in background

### ISSUE THREAD

**Issue by cheng-tan** [covidsafe/App-iOS#116]
stacktrace:
```
Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Triggered by Thread:  2

Last Exception Backtrace:
0   CoreFoundation                	0x187160164 __exceptionPreprocess + 228 (NSException.m:199)
1   libobjc.A.dylib               	0x186e74c1c objc_exception_throw + 60 (objc-exception.mm:565)
2   CoreFoundation                	0x18704fbb8 +[NSException raise:format:] + 112 (NSException.m:155)
3   covidsafe                     	0x100d68e44 -[BLE init_module::options:] + 2028 (BLE.mm:494)
4   CoreFoundation                	0x187166d14 __invoking___ + 148
5   CoreFoundation                	0x18702fde8 -[NSInvocation invoke] + 400 (NSForwarding.m:3312)
6   CoreFoundation                	0x187030a10 -[NSInvocation invokeWithTarget:] + 80 (NSForwarding.m:3418)
7   covidsafe                     	0x100e3d314 -[RCTModuleMethod invokeWithBridge:module:arguments:] + 460 (RCTModuleMethod.mm:569)
8   covidsafe                     	0x100e3f400 facebook::react::invokeInner(RCTBridge*, RCTModuleData*, unsigned int, folly::dynamic const&) + 244 (RCTNativeModule.mm:108)
9   covidsafe                     	0x100e3f164 invocation function for block in facebook::react::RCTNativeModule::invoke(unsigned int, folly::dynamic&&, int) + 88 (RCTNativeModule.mm:73)
10  libdispatch.dylib             	0x186dfeec4 _dispatch_call_block_and_release + 32 (init.c:1408)
11  libdispatch.dylib             	0x186e0033c _dispatch_client_callout + 20 (object.m:495)
12  libdispatch.dylib             	0x186e0685c _dispatch_lane_serial_drain + 568 (inline_internal.h:2484)
13  libdispatch.dylib             	0x186e07290 _dispatch_lane_invoke + 400 (queue.c:3863)
14  libdispatch.dylib             	0x186e10928 _dispatch_workloop_worker_thread + 584 (queue.c:6445)
15  libsystem_pthread.dylib       	0x186e67714 _pthread_wqthread + 276 (pthread.c:2351)
16  libsystem_pthread.dylib       	0x186e6d9c8 start_wqthread + 8
```

related to #115

**Comment by cheng-tan** [comment]
Need to test if this is fixed with #118

**Comment by cheng-tan** [comment]
verified with build 6 not seeing this error anymore, closing.
